### PR TITLE
Moving the setting of the classifier path before initializing the cla…

### PR DESCRIPTION
**Pull request by umangmehta12**
…ssifiers

### CODE REVIEW

File: opencv-webcam-demo/opencv-webcam-demo.cpp [hunk 1]
```diff
@@ -113,11 +113,11 @@ int main(int argsc, char ** argsv)
         frameDetector = make_shared<FrameDetector>(buffer_length, process_framerate, nFaces, (affdex::FaceDetectorMode) faceDetectorMode);        // Init the FrameDetector Class
 
         //Initialize detectors
-        frameDetector->setDetectAllEmotions(true);
+        frameDetector->setClassifierPath(DATA_FOLDER);
+	frameDetector->setDetectAllEmotions(true);
```
Comment (adennie) [inline opencv-webcam-demo/opencv-webcam-demo.cpp]: looks like a tab slipped in here, throwing off the indentation

File: video-demo/video-demo.cpp [hunk 2]
```diff
@@ -135,11 +135,11 @@ int main(int argsc, char ** argsv)
         std::cout << "Face detector mode set to: " << mode << std::endl;
         shared_ptr<PlottingImageListener> listenPtr(new PlottingImageListener(csvFileStream, draw_display));
 
+	detector->setClassifierPath(DATA_FOLDER);
```
Comment (adennie) [inline video-demo/video-demo.cpp]: tab

File: video-demo/video-demo.cpp [hunk 3]
```diff
@@ -135,11 +135,11 @@ int main(int argsc, char ** argsv)
...
         detector->setDetectAllEmotions(true);
         detector->setDetectAllExpressions(true);
-		detector->setDetectAllEmojis(true);
+	detector->setDetectAllEmojis(true);
```
Comment (adennie) [inline video-demo/video-demo.cpp]: tab

File: opencv-webcam-demo/opencv-webcam-demo.cpp (hunk 1 above)
Comment (umangmehta12) [reply opencv-webcam-demo/opencv-webcam-demo.cpp]: Perks of using vim @adennie, I fixed this!

**Pull request merged**